frontend/src/main/scala/akka/viz/frontend/components/components.scala

```diff
@@ -183,6 +184,44 @@ class MessageFilter(
 }
 
 class MessagesPanel(selectedActors: Var[Set[String]]) extends Component with PrettyJson {
+  val ShowMoreLength = 200
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Field name does not match the regular expression '^[a-z][A-Za-z0-9]*$'](https://www.codacy.com/app/lustefaniak/akka-viz/file/2015979973/issues/source?bid=3121282&fileBranchId=3123640#l187)